install_static_swift_modules/after/Pods/Headers/Public/MixedPod/MixedPod-library-macOS.modulemap

```diff
@@ -1 +1 @@
-/Users/segiddins/Development/Rainforest/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/MixedPod-library-macOS/MixedPod-library-macOS.modulemap
\ No newline at end of file
+/Users/derko/Development/cocoapods/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/MixedPod-library-macOS/MixedPod-library-macOS.modulemap
```

Comment:
uh why do we have the absolute path here?

Reply:
https://cocoapods.slack.com/archives/C029NJ045/p1533581255000320